tests/schema/models.py

```diff
@@ -186,7 +187,7 @@ class TagIndexed(models.Model):
 
     class Meta:
         apps = new_apps
-        index_together = [["slug", "title"]]
+        indexes = [models.Index(fields=["slug", "title"])]
```

Comment:
This model is used in `test_create_index_together()` so we shouldn't change `index_together` to `indexes`. I moved into the test method in #15754.